rules/linux/credential_access_shadow_file_access.toml

```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2022/09/01"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/11/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies access to the /etc/shadow file via the commandline using standard system utilities. After elevating
+privileges to root, threat actors may attempt to read or dump this file in order to gain valid credentials. They may
+utilize these to move laterally undetected and access additional resources.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Shadow File Read via Command Line Utilities"
+references = ["https://www.cyberciti.biz/faq/unix-linux-password-cracking-john-the-ripper/"]
+risk_score = 47
+rule_id = "9a3a3689-8ed1-4cdb-83fb-9506db54c61f"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and 
+  event.action == "exec" and user.name == "root" and
+  (process.args : "/etc/shadow" or (process.working_directory: "/etc" and process.args: "shadow")) 
+  and not process.executable: ("/usr/bin/find", "/usr/bin/cmp", "/bin/ls", "/usr/sbin/restorecon", "/usr/bin/uniq") 
```

Comment:
I would suggest we look at tele to determine if this is noisy and see any addition exclusions can be added as part of the same PR. 

This seems fairly noisy.
For example executables "/usr/bin/cmp" is being excluded but executables like "/bin/cmp" is being flagged. Exclusion criteria does not include grep very similar to find.

Can be further tuned to reduce noise

Reply:
I agree, will look at telemetry for more potential exclusions